procedures/katello/branch.md.erb

```diff
@@ -64,10 +64,10 @@
 
 ## Release Owner
 
-- [ ] Create KATELLO-<%= release %> branches
-  - [ ] [katello](https://github.com/katello/katello)
-- [ ] Bump versions to <%= develop %>-master
-  - [ ] [katello](https://github.com/katello/katello)
+- [ ] Create KATELLO-<%= release %> branches using [branch_project](https://github.com/theforeman/theforeman-rel-eng/blob/master/branch_project)
```

Comment:
I would change `branches` to `branch`

Reply:
Oh yes, for Katello it's one branch. In Foreman the same script does it for multiple branches.